### DISCUSSION TITLE
Spool Manager

### DISCUSSION THREAD

**Discussion by champion0369** [UnchartedBull/OctoDash#2240] [Q&A]
I am trying to get spool manager to work with octodash. The addon doesn't even show up inside the gui addon found in octodash but I suspected that it would only be available at install. I used remove.sh to clear octodash and reinstalled. I did indeed get an option to install spool manager.  Once I got into octoprint I found that filament manager was actually installed and would work with the filament manager plugin in octoprint. No go for spool manager. Is there anything I am doing wrong? I know this is a new feature, but I am super excited since spool manager should be more user friendly than filament manager.

**Comment by UnchartedBull** [comment]
Filament Manager will be released with the next version. If you want to try a nightly build with Filament Manager already enabled please download that from here: https://github.com/jneilliii/OctoDash/actions/runs/1172339904

**Comment by mhegering** [comment]
Hey @UnchartedBull, im already waiting for the next release because i want to use SpoolManager so much.
Can you tell me when the new release will come? 
If i could i would build it all by myself but i'm able to do that..

**Reply by jneilliii** [reply]
UnchartedBull linked the build that has it already. You just need to download it from there.

**Reply by mhegering** [reply]
Oh really, i have looked everywhere couldn't find it..

**Reply by jneilliii** [reply]
Looks like that build expired. The latest build that has those previously mentioned changes and the latest notification center (swipe down from top for custom actions, clock) can be found here. https://github.com/jneilliii/OctoDash/actions/runs/1527977960 download the artifact, extract, transfer the arm.deb file to your pi using winscp or filezilla ftp client and then install using the command `sudo dpkg -i ./filename.deb` and then reboot.

**Reply by mhegering** [reply]
Thank you very much!
